mlir/test/Conversion/VectorToLLVM/vector-to-llvm.mlir

```diff
@@ -1819,16 +1819,55 @@ func.func @genbool_1d() -> vector<8xi1> {
 
 // -----
 
-func.func @genbool_1d_scalable() -> vector<[8]xi1> {
```

Comment:
[nit] `ptrue` is a very "SVE term", not sure about `pfalse` tbh. Perhaps `all_true` and `all_false` instead? That would be more generic.